chart/jenkins-operator/Chart.yaml

```diff
@@ -2,5 +2,5 @@ apiVersion: v2
 appVersion: "0.6.0"
 description: Kubernetes native operator which fully manages Jenkins on Kubernetes
 name: jenkins-operator
-version: 0.5.1
+version: 0.5.2
```

Comment:
This can be omitted, we (maintainers) will take care of the releasing process

Reply:
Should I revert the commits which bumb chart version?

Reply:
I think if you'd merge/rebase changes from currently updated master there'd be no diff in that file since it was bumped today

Reply:
I think if you'd merge/rebase changes from currently updated master there'd be no diff in that file since it was bumped today

Reply:
@Sig00rd , so what should I do now? Is there anything I should/can add to this MR to have it accepted?